.dockerignore

```diff
@@ -79,9 +80,8 @@ tests/cell_fabric/Foo*.cir
 !docs/Tutorials/Build/src/json/
 !tests/cell_fabric/__json_cmc_nmos_big_no_duplicates_cand
 !tests/cell_fabric/__json_diff_pair_cand
-!tests/compiler/test_results/test_cap.const.json
-!tests/compiler/test_circuits/high_speed_comparator_*/high_speed_comparator.const.json
-!tests/compiler/test_results/high_speed_comparator_*.const.json
+!tests/compiler/test_circuits/high_speed_comparator_*/high_speed_comparator.pnr.const.json
```

Comment:
I verified that we need to keep the .dockerignore consistent with the .gitignore (as you have done here).